src/Components/Patient/FileUpload.tsx

```diff
@@ -247,6 +249,10 @@ export const FileUpload = (props: FileUploadProps) => {
         limit: limit,
         offset: offset,
       };
+      const patientDetails = await dispatch(getPatient({ id: patientId }));
+      if (patientDetails.data) {
+        setIsActive(patientDetails.data.is_active);
+      }
```

Comment:
we are already calling `getPatient` in line 163, so getting the `is_active` attribute from that would reduce the extra API call

Reply:
Done